applications/connecting_applications_to_services/binding-workloads-using-sbo.adoc

```diff
@@ -0,0 +1,47 @@
+[id="binding-workloads-using-sbo"]
+= Binding workloads using Service Binding Operator
+include::modules/common-attributes.adoc[]
+include::modules/servicebinding-document-attributes.adoc[]
+:context: binding-workloads-using-sbo
+
+toc::[]
+
+Application developers must bind a workload to one or more backing services using a binding secret generated for the purpose of storing information to be consumed by the workload.
+
+As an example, consider that the service you want to connect to is already exposing the binding data. You also need a workload to be used along with the `ServiceBinding` custom resource (CR). Through this `ServiceBinding` CR the workload sends a binding request with the details of the services to bind with.
```

Comment:
```suggestion
As an example, consider that the service you want to connect to is already exposing the binding data. In this case, you would also need a workload to be used along with the `ServiceBinding` custom resource (CR). Bu using this `ServiceBinding` CR, the workload sends a binding request with the details of the services to bind with.
```
- Use the conditional tense here for a hypothetical example.
- A couple of other small suggestions